content/docs/command-reference/run.md

```diff
@@ -6,10 +6,12 @@ command and execute the command.
 ## Synopsis
 
 ```usage
-usage: dvc run [-h] [-q | -v] [-d <path>] [-o <path>] [-O <path>]
-               [-m <path>] [-M <path>] [-f <filename>] [-c <path>]
-               [-w <path>] [--no-exec] [-y] [--overwrite-dvcfile]
+usage: dvc run [-h] [-q | -v] [-d DEPS] [-o OUTS] [-O OUTS_NO_CACHE]
+               [-p PARAMS] [-m METRICS] [-M METRICS_NO_CACHE] [-f FILE]
+               [-c CWD] [-w WDIR] [--no-exec] [-y] [--overwrite-dvcfile]
                [--ignore-build-cache] [--remove-outs] [--no-commit]
+               [--outs-persist OUTS_PERSIST]
+               [--outs-persist-no-cache OUTS_PERSIST_NO_CACHE]
                [--always-changed]
                command
```

Comment:
One lat comment: `dvc run` is getting gargantuan 🐋

Isn't there an issue out there somewhere to split it into 2 commands? May be time to revisit that 😬